File: bootstrap/src/main/java/net/dontdrinkandroot/wicket/bootstrap/css/BadgeStyle.java
package net.dontdrinkandroot.wicket.bootstrap.css;

import net.dontdrinkandroot.wicket.css.CssClass;


public enum BadgeStyle implements CssClass {

	IMPORTANT("badge-important"),
	INFO("badge-info"),
	INVERSE("badge-inverse"),
	SUCCESS("badge-success"),
	WARNING("badge-warning");

	private String classString;


	private BadgeStyle(String classString) {

		this.classString = classString;
	}


	@Override
	public String getClassString() {

		return this.classString;
	}
}

File: bootstrap/src/main/kotlin/net/dontdrinkandroot/wicket/bootstrap/css/BadgeStyle.kt
package net.dontdrinkandroot.wicket.bootstrap.css

import net.dontdrinkandroot.wicket.css.CssClass

enum class BadgeStyle(override val classString: String) : CssClass {
    PRIMARY("badge-primary"),
    SECONDARY("badge-secondary"),
    SUCCESS("badge-success"),
    DANGER("badge-danger"),
    WARNING("badge-warning"),
    INFO("badge-info"),
    LIGHT("badge-light"),
    DARK("badge-dark");
}